Completion/Unix/Command/_notmuch

```diff
@@ -1,80 +1,292 @@
-#compdef notmuch
+#compdef notmuch -p notmuch-*
 
-_notmuch_commands()
-{
+# ZSH completion for `notmuch`
+# Copyright © 2018 Vincent Breitmoser <look@my.amazin.horse>
```

Comment:
Under what license does Mr Breitmoser place the code?

Reply:
Notmuch is GPL-3, based on the COPYING file in the project root [0]. I'm only assuming this applies to the zsh completions as well

https://github.com/notmuch/notmuch/blob/master/COPYING

Reply:
We don't just pull in GPL-3 licenced code into zsh, even if it is in a self-contained file like a completion function. While we can do this in theory, it forces a big pile or bureaucratic work on packagers for some distributions if zsh comes under a mix of licences. If notmuch includes a zsh completion, we should simply remove the _notmuch from the zsh distribution.

Reply:
I could also delete this PR and send a new one just removing these files from the zsh distribution